.gitignore

```diff
@@ -8,8 +8,7 @@
 /bin/
 /examples/bin/
 /lib/
-# Travis CI build directory
-/build/
+
```

Comment:
Removing this in light of the new build instructions, as this "ignore" makes it less obvious to a user that they have a build directory floating around.

Happy to change it back if folks disagree, but I find it helpful when `git status` shows me what new files have been created by the build.

Reply:
I disagree with this change, this seems to be opposite the usual Git workflow of considering build files to not be tracked by version control. Most of the paths listed here are specifically for the build process.

Reply:
That makes sense. I'll change it back.